docs/reqparse.rst

```diff
@@ -226,20 +226,19 @@ then the error message will be the value of ``help``.
 
 ``help`` may include an interpolation token, ``{error_msg}``, that will be
 replaced with the string representation of the type error. This allows the
-message to be customized while preserving the original error:
+message to be customized while preserving the original error ::
 
     from flask_restful import reqparse
-
-
+    
     parser = reqparse.RequestParser()
     parser.add_argument(
         'foo',
         choices=('one', 'two'),
         help='Bad choice: {error_msg}'
     )
-
+    
```

Comment:
Actually there's no trailing spaces in the last line if you edit with Github's editor. I'm not sure why it's showing a difference with spaces.